HyperwalletUISDK.podspec

```diff
@@ -6,18 +6,18 @@ Pod::Spec.new do |spec|
     spec.license               = { :type => 'MIT', :file => 'LICENSE' }
```

Comment:
You will have to update `spec.version               = '1.0.0-beta16'` as well

Reply:
it's already updated https://github.com/hyperwallet/hyperwallet-ios-ui-sdk/blob/f9901ebd6dc44d9e3db8ebe1335ca8bab2fe7271/HyperwalletUISDK.podspec

Reply:
I am talking about the new version that you will release of UI SDK. You need to update the new version for cocoapods as well

Reply:
got it, updated to version beta 17